src/resources/views/auth/fullpage_redirect.blade.php

```diff
@@ -4,7 +4,6 @@
         <meta charset="utf-8">
         <base target="_top">
         <meta name="shopify-api-key" content="{{ \Osiset\ShopifyApp\Util::getShopifyConfig('api_key', $shopDomain ?? Auth::user()->name ) }}"/>
-        <script src="https://cdn.shopify.com/shopifycloud/app-bridge.js"></script>
 
         <title>Redirecting...</title>
 
```

Comment:
why do we need this view? it works if we make the redirect from the controller without using the view.

Reply:
As I mentioned in the issue, there is no difference; both options will likely work the same way.

However, someone might be using this view to send analytics via sendBeacon or for something else. I can't make any definitive statements here.

Personally, I don't use the authenticate route at all—it's not necessary during a token exchange.

Reply:
This is from memory but...

Full page redirect view is for after the /authenticate route runs, this full page redirect is what moves you to to auth screen on Shopify to accept the install.